src/Infiltration/formulas/victory.ts

```diff
@@ -45,7 +45,7 @@ export function calculateTradeInformationRepReward(
   }
 
   return (
-    Math.pow(reward, 1.1) *
+    Math.pow(reward + Math.min(1, reward / 2), 1.1) *
```

Comment:
I didn't even realize this is how reward gets mixed in. This makes it a lot more complicated to figure out the optional speed of auto infil (for those inclined to do such a thing) - the calculus gets much worse. Why not pass market demand in here and just have it be a linear factor?